### PR TITLE
Remove SEAMLESS mention in bootstrap

### PR DESCRIPTION
**Pull request by sneridagh**
- [ ] I signed and returned the [Plone Contributor Agreement](https://plone.org/foundation/contributors-agreement), and received and accepted an invitation to join a team in the Plone GitHub organization.
- [ ] I verified there aren't other open [pull requests](https://github.com/plone/volto/pulls) for the same change.
- [ ] I followed the guidelines in [Contributing to Volto](https://6.docs.plone.org/volto/contributing/index.html).
- [ ] I succesfully ran [code linting checks](https://6.docs.plone.org/volto/contributing/linting.html) on my changes locally.
- [ ] I succesfully ran [unit tests](https://6.docs.plone.org/volto/contributing/testing.html) on my changes locally.
- [ ] I succesfully ran [acceptance tests](https://6.docs.plone.org/volto/contributing/acceptance-tests.html) on my changes locally.
- [ ] If needed, I added new tests for my changes.
- [ ] If needed, I added [documentation](https://6.docs.plone.org/volto/contributing/documentation.html#narrative-documentation) for my changes, either in the Storybook or narrative documentation.
- [ ] I included a [change log entry](https://6.docs.plone.org/contributing/index.html#contributing-change-log-label) in my commits.

-----

If your pull request closes an open issue, include the exact text below, immediately followed by the issue number. When your pull request gets merged, then that issue will close automatically.

Closes #

### CODE REVIEW

File: packages/volto/src/start-server.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ export default function server() {
   server
     .listen(port, bind_address, () => {
       if (app.apiPath === app.publicURL || !app.apiPath) {
-        console.log(`Volto is running in SEAMLESS mode`);
+        console.log(`Volto is running in normal deployment mode`);
       } else {
         console.log(`API server (API_PATH) is set to: ${app.apiPath}`);
```
Comment (davisagli) [inline packages/volto/src/start-server.js:24]: Maybe it would be clearer to remove the condition altogether and always log line 24?

**Review by davisagli — COMMENTED** [review]
Maybe it wou

File: packages/volto/news/6424.internal [hunk 2]
```diff
@@ -0,0 +1 @@
+Remove mention to "SEAMLESS" mode in the server bootstrap @sneridagh
```
Comment (stevepiercy) [inline packages/volto/news/6424.internal:1]:
```suggestion
Remove mention of `SEAMLESS` mode in the server bootstrap. @sneridagh
```

File: packages/volto/src/start-server.js (hunk 1 above)
Comment (stevepiercy) [reply packages/volto/src/start-server.js:24]: @davisagli remove the `else` condition only, or both the `if` and `else` conditions?

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Minor change to news, else I approve, pending resolution of @davisagli's comment.

**Comment by davisagli** [comment]
@stevepiercy I'm suggesting to keep this line only and always log the API URL: `console.log(`API server (API_PATH) is set to: ${app.apiPath}`);`

File: packages/volto/src/start-server.js (hunk 1 above)
Comment (sneridagh) [reply packages/volto/src/start-server.js:24]: Yeah, I thought that the condition might be useful, but it's clear that it's not.

**Comment by sneridagh** [comment]
@davisagli @stevepiercy done!

**Review by davisagli — APPROVED** [review]

**Pull request merged**